examples/snaps/find-output.json

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
How is this generated?  The help section for the --snaps-from-examples could mention that this loads output from 'snap find output' or whatever is needed to fetch this data in this format.

Reply:
I added a examples/snaps/README.md file and mentioned that file in the help.